aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -653,20 +677,30 @@ module aptos_framework::stake {
             error::invalid_argument(EALREADY_ACTIVE_VALIDATOR),
         );
 
-        // Requires the validator to have a lockup more than the minimum.
         let config = staking_config::get();
         let (minimum_stake, maximum_stake) = staking_config::get_required_stake(&config);
-
         // The validator is not yet activated so all added stake should be in active.
         let voting_power = coin::value<AptosCoin>(&stake_pool.active);
```

Comment:
this is probably not accurate, a left validator with pending_inactive can rejoin? we need to update the comment as well

Reply:
Updated to be sum of pending_active (activated next epoch) + active + pending_inactive (might still be around next epoch if lockup has not expired).